docs/src/layout/page/demo/demo.jsx

```diff
@@ -208,6 +208,25 @@ class PageDemo extends React.PureComponent {
             onClearClick: this.onSearchClear,
             onChange: this.onSearchChange,
             onKeyDown: this.onSearchKeyDown,
+            searchWithSelect: {
+                onChange: _.noop,
```

Comment:
NAB: _Eventually_ let's update this to do separate Lodash function imports,  It's important to do this even for example code, since people might follow, and even copy and paste from examples to live code! =)

But does not have to necessarily be done in this PR; just flagging it for the future.

Reply:
Copy/Paste on live code =) makes sense. I am on it. Thank you